src/libvirtApi/snapshot.ts

```diff
@@ -144,7 +144,7 @@ export async function snapshotGetAll({
         store.dispatch(updateDomainSnapshots({
             connectionName,
             domainPath,
-            snaps: undefined,
+            snaps: false,
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2386-cd287e64-20251030-080645-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)